core/src/display_object/avm2_button.rs

```diff
@@ -620,8 +620,17 @@ impl<'gc> TDisplayObject<'gc> for Avm2Button<'gc> {
     }
 
     fn bounds_with_transform(&self, matrix: &Matrix) -> Rectangle<Twips> {
-        // [NA]: The diff between this and the base impl is a lack of scroll_rect.
-        // Intentional, or bug?
+        // A scroll rect completely overrides an object's bounds,
+        // and can even the bounding box to be larger than the actual content
```

Comment:
nit: grammatical mistake "and can even the bounding box"
(unless "even" is being used as a verb, but that also seems wrong)

Reply:
Literally just copy pasted from the super method, didn't even read it 😅

I can change them both